### ISSUE TITLE
Add a config parameter to disable console output

### ISSUE THREAD

**Issue by divinites** [LuckyGeck/YcmdCompletion#21]
This is a very helpful plugin, Thank you very much. It is a bit too talktative in the console. I understand that these messages are very helpful when configuring and debugging. But when the user is sure that everything is okay, It would nice if there exists a config parameter that can let ycmd work in a silent mode.

I discovered that most of the messages are generated by print_status(), which basically is a print.  probably you can add an additional boolean arg to print_status, which makes this function print if and only if this arg == true.  Just my thought, probably you can find better solution that this.

Thanks.

**Comment by LuckyGeck** [comment]
Yeah, we surely can disable debug output. But I don't see any profit for a regular user - you don't always see this console at all.

And yes - PR's are always welcome. You can add a config option (switched off by default), that disables console messages.
